src/content/test/semantics/headers-attribute.tsx

```diff
@@ -6,8 +6,8 @@ export const infoAndExamples = create(({ Markup, Link }) => (
     <>
         <h1>Headers attribute</h1>
         <p>
-            The <Markup.Term>headers</Markup.Term> attribute of a <Markup.Term>{`<td>`}</Markup.Term> element must reference the correct{' '}
-            <Markup.Term>{`<th>`}</Markup.Term> element(s).
+            The <Markup.Code>headers</Markup.Code> attribute of a <Markup.Code>{`<td>`}</Markup.Code> element must reference the correct{' '}
```

Comment:
Should the td/th references in here be using `<Markup.Tag tagName="whatever" />`?

Reply:
We aren't using Tag in any info and example content. Did not use it for consistency.